backend/routes/ContactDetails.ts

```diff
@@ -0,0 +1,91 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import fs from 'fs';
+import { getLogger } from '../handler/LoggerHandler';
+import { getResourcePath } from '../utils';
+import { jsonRequest } from '../handler/ManagerHandler';
+
+const logger = getLogger('ContactDetails');
+const router = express.Router();
+const contactDetailsFilePath = getResourcePath('submittedContactDetails.json', true);
+
+/* eslint-disable camelcase */
+// NOTE: DTO are used as a shapes of requests which are sent/received
+interface ContactDetails {
+    first_name: string;
+    last_name: string;
+    email: string;
+    phone: string;
+    is_eula: boolean;
+    is_send_services_details?: boolean;
+}
+
+interface HubspotResponse {
+    customer_id: string;
+}
+/* eslint-enable camelcase */
+
+// Reading the data from the stored file (which definition is described with the usage of StoredContactDetails)
+const getStoredContactDetails = (): ContactDetails => {
```

Comment:
There's small misalignment with naming I think as we don't have anything like `StoredContactDetails` or am I missing something?

Maybe we should have here:
```suggestion
// Reading the data from the stored file (which definition is described with the usage of ContactDetails)
const getContactDetails = (): ContactDetails => {
```
?

Reply:
I believe the term 'stored' in the function name refers to reading `ContactDetails` instance from a file as opposed to getting it from the request, which is fine and accurate. I updated just the comment.